docs/contextual-bandits/deployment.md

```diff
@@ -0,0 +1,74 @@
+---
+sidebar_position: 2
+---
+# Creation and Deployment
+
+bandit is deployed as part of a feature flag. this also enables experiment analysis.
```

Comment:
Should this be capitalized?

Reply:
Good catch, those were some initla notes -- removed